tests/Generating/BasicGeneratorTest.cpp

```diff
@@ -245,6 +256,11 @@ static void checkChunkChecksums(
 	{
 		cChunkDesc chd(coords.mCoords);
 		aGenerator.Generate(chd);
+		/*
+		cFile f(Printf("Repeatability_%s-%02d-%02d.raw", aDimension, coords.mCoords.m_ChunkX, coords.mCoords.m_ChunkZ), cFile::fmWrite);
+		f.Write(chd.GetBlockTypes(), sizeof(chd.GetBlockTypes()));
+		f.Close();
```

Comment:
Is this intended to remain?

Reply:
Yes, it is needed for debugging whenever this test fails, so why not keep it in, although commented-out.